### PR TITLE
ovl_Oceff_Wipe7 OK

### PR DESCRIPTION
**Pull request by sonicdcer**

### CODE REVIEW

**Review by jpburnett — APPROVED** [review]

File: src/overlays/actors/ovl_Oceff_Wipe7/z_oceff_wipe7.c [hunk 1]
```diff
@@ -28,12 +27,79 @@ const ActorInit Oceff_Wipe7_InitVars = {
...
+    f32 z;
+    u8 alpha;
+    s32 i;
+    s32 counter;
+    Vec3f activeCamEye;
+    s32 pad;
+    Vec3f quakeOffset;
+    s32 pad2;
+
+    activeCamEye = GET_ACTIVE_CAM(play)->eye;
```
Comment (hensldm) [inline src/overlays/actors/ovl_Oceff_Wipe7/z_oceff_wipe7.c:71]:
```suggestion
    Vec3f activeCamEye;
    s32 pad;
    Vec3f  eye = GET_ACTIVE_CAM(play)->eye;
    s32 pad2;

```

File: src/overlays/actors/ovl_Oceff_Wipe7/z_oceff_wipe7.c [hunk 2]
```diff
@@ -28,12 +27,79 @@ const ActorInit Oceff_Wipe7_InitVars = {
...
+    Camera_GetQuakeOffset(&quakeOffset, GET_ACTIVE_CAM(play));
+
+    if (this->counter < 32) {
+        counter = this->counter;
+        z = Math_SinS(counter * 0x200) * 1220.0f;
```
Comment (hensldm) [inline src/overlays/actors/ovl_Oceff_Wipe7/z_oceff_wipe7.c:76]:
```suggestion
        z = Math_SinS(this->counter * 0x200) * 1220.0f;
```

**Review by AngheloAlf — APPROVED** [review]

File: assets/xml/overlays/ovl_Oceff_Wipe7.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<Root>
+    <File Name="ovl_Oceff_Wipe7" BaseAddress="0x80BCDCB0" RangeStart="0x330" RangeEnd="0xDB8">
+        <Texture Name="gOceff7Tex" OutName="oceff7" Format="i8" Width="32" Height="32" Offset="0x330"/>
+        <TextureAnimation Name="ovl_Oceff_Wipe7_Matanimheader_000B38" Offset="0xB38" />
+        <Array Name="gOceff7Vtx" Count="22" Offset="0xB40">
+            <Vtx/>
+        </Array>
+        <DList Name="gOceff7DL" Offset="0xCA0"/>
```
Comment (hensldm) [inline assets/xml/overlays/ovl_Oceff_Wipe7.xml:8]:
```suggestion
        <Texture Name="sSongOfHealingEffectTex" OutName="song_of_healing_effect" Format="i8" Width="32" Height="32" Offset="0x330"/>
        <TextureAnimation Name="sSongofHealingEffectTexAnim" Offset="0xB38" />
        <Array Name="sSongofHealingEffectFrustrumVtx" Count="22" Offset="0xB40">
            <Vtx/>
        </Array>
        <DList Name="sSongOfHealingEffectFrustumDL" Offset="0xCA0"/>
```

File: assets/xml/overlays/ovl_Oceff_Wipe7.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<Root>
+    <File Name="ovl_Oceff_Wipe7" BaseAddress="0x80BCDCB0" RangeStart="0x330" RangeEnd="0xDB8">
+        <Texture Name="gOceff7Tex" OutName="oceff7" Format="i8" Width="32" Height="32" Offset="0x330"/>
+        <TextureAnimation Name="ovl_Oceff_Wipe7_Matanimheader_000B38" Offset="0xB38" />
+        <Array Name="gOceff7Vtx" Count="22" Offset="0xB40">
+            <Vtx/>
+        </Array>
```
Comment (hensldm) [inline assets/xml/overlays/ovl_Oceff_Wipe7.xml:7]: Looks like you should be fine just extracting these implicitly.

**Review by Kenix3 — APPROVED** [review]

File: assets/xml/overlays/ovl_Oceff_Wipe7.xml (hunk 4 above)
Comment (sonicdcer) [reply assets/xml/overlays/ovl_Oceff_Wipe7.xml:7]: it is used in the draw function

**Pull request merged**